Tests/SwiftGenTests/ConfigLintTests.swift

```diff
@@ -83,15 +84,17 @@ class ConfigLintTests: XCTestCase {
       [
         (.warning, """
           \(cmd).inputs: /input/\(cmd)/paths is an absolute path. Prefer relative paths for \
-          portability when sharing your project.
+          portability when sharing your project (unless you are using environment variables).
           """),
         (.warning, """
           \(cmd).outputs.templatePath: /templates/\(cmd)/templates.stencil is an absolute path. \
-          Prefer relative paths for portability when sharing your project.
+          Prefer relative paths for portability when sharing your project (unless you are using \
+          environment variables).
           """),
         (.warning, """
           \(cmd).outputs.output: /output/\(cmd)/out.swift is an absolute path. Prefer relative \
-          paths for portability when sharing your project.
+          paths for portability when sharing your project (unless you are using environment \
+          variables).
```

Comment:
Wondering if it would be worth putting that expected error message (well the common part starting from "is an absolute part…") in an intermediate constant?! 🤔

Reply:
bcf1eb8 what you had in mind?

Reply:
Wow didn't expect to go that far, was just thinking of a plain string `let`… but yeah seems great 👍